styles.css

```diff
@@ -695,13 +833,33 @@ img.logo-img {
     color: #d3d3d3;
   }
 
+  .sponsortext {
+    font-size: x-large;
+    font-weight: 500 !important;
+    font-family: "Poppins", sans-serif;
+    color: #d3d3d3;
+  }
+
   .partners {
     background-color: rgb(39, 42, 50);
     display: flex;
     flex-direction: column;
     gap: 30px;
   }
```

Comment:
Great design @godybuoy 🤝

- [ ] Kindly ensure your partners' section has five partners so that your design may look exactly like the design given.
- Your design
![Your-partners](https://user-images.githubusercontent.com/63917725/235325055-76ccd5c8-1321-4c3a-a896-582cf29f3400.PNG)
- Given design
![Given-partners](https://user-images.githubusercontent.com/63917725/235325066-cc770d98-e320-4717-88c1-fdfa053517d4.PNG)